docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -0,0 +1,428 @@
+
+- Do not use symbols and special characters in headings. 
+
+- When possible, avoid including words that might change in the future. Changing a heading changes its anchor URL, which affects other pages that link to this headline.
+
+- Leave exactly one blank line before and after a heading.
+
+- Do not use links in headings.
+
+- Make your subheading titles clear, descriptive, and complete to help users find the right example.
+
+- See  [Capitalization](../#capitalization) for guidelines on capitalizing headings.
+
+  
+
+### Heading titles
+
+Keep heading titles clear and direct. Make every word count. Where possible, use the imperative. Example: Sign in with an existing account (**not** Signing in with an existing account).
+
+
+
+### Anchor links
+
+Headings generate anchor links when rendered. ##This is an example generates the anchor #this-is-an-example.
+
+Keep in mind that there are various links to OpenProject documentation pages and anchor links on the internet to take the users to the right spot. Thus, please avoid changing headings.
+
+
+
+## Links
+
+Links are important in the documentation. Use links instead of duplicating content to help preserve a single source of truth in the OpenProject documentation.
+
+
+
+### Basic link criteria
+
+- Use inline link Markdown markup `[Description](https://example.com)`. It is easier to read, review, and maintain.
+
+- Use meaningful anchor text descriptions. For example, instead of writing something like `Read more about Gantt charts [here](LINK)`, write `Read more about [Gantt charts](LINK)`.
+
+
+
+### Links to internal documentation
+
+Internal links are links within the OpenProject website which includes the OpenProject documentation. In these cases, use relative links. I.e. do not use the full URL of the linked page but instead show the current URL's relation to the linked page's URL. 
+
+To link to internal documentation:
+
+- Use relative links to Markdown files in the same repository.
+- Use ../ to navigate to higher-level directories.
+
+
+
+### Links to external documentation
+
+When linking to external information, you have to use absolute URLs. Make sure that you are only linking to an authoritative source, i.e. official and credible sources written by the people who created the item or product. These sources are the most likely to be accurate and remain up to date.
+
+ 
+
+## Navigation
+
+When documenting navigation through the OpenProject application, use these terms.
+
+
+
+### Menus
+
+Use these terms when referring to OpenProject’s main application elements:
+
+- **Header menu**: This is the blue bar at the top that spans the width of the application. It includes the OpenProject logo, the search field, the link to all projects, the short menu, the help icon and the user’s avatar.
+
+- **Project menu**: This is the black menu on the left in the OpenProject application that displays the modules.
```

Comment:
Does not necessarily have to be black. Maybe just the menu on the left that appears after the selection of a project.